pkgs/applications/version-management/josh/default.nix

```diff
@@ -17,17 +17,18 @@ in
 
 rustPlatform.buildRustPackage rec {
   pname = "josh";
-  version = "23.12.04";
-  JOSH_VERSION = "r${version}";
+  # No release yet including Rust 1.80 fix:
+  # https://github.com/josh-project/josh/issues/1353#issuecomment-2286168249
+  version = "23.12.04-unstable-2024-08-15";
```

Comment:
Upstream just did tag this commit as https://github.com/josh-project/josh/releases/tag/v24.08.14